tools/cli/NearObjectCli.cxx

```diff
@@ -211,7 +211,12 @@ CLI::Option*
 AddEnumOption(CLI::App* app, std::optional<EnumType>& assignTo, bool isMandatory = false)
 // clang-format on
 {
-    std::string optionName{ std::string("--").append(magic_enum::enum_type_name<EnumType>()) };
+    std::string optionName{ std::string("--") };
+    if (magic_enum::enum_type_name<EnumType>() == "StsPacketConfiguration") {
+        optionName.append("RFrameConfiguration");
```

Comment:
Instead of this, is it possible to alias the enum? Eg. Add this to `FiraDevice.hxx` after the declaration of `enum class StsPacketConfiguration ... { `

`using RFrameConfiguration = StsPacketConfiguration;`

Reply:
It may be possible. I vaguely remember something in the docs for magic_enum that it doesn't work with aliasing, but I will give it a try!

Reply:
The alternative would be to rename the enum, then introduce an alias for the current name so that it can continue to be used as-is, but you can use `RFrameConfiguration` in the CLI.

Reply:
Yeah unfortunately I can confirm that magic_enum doesn't recognize the alias, so I will try your second suggestion - rename the enum to `RFrameConfiguration` and add an alias like `using StsPacketConfiguration = RFrameConfiguration`